include/mega/posix/megawaiter.h

```diff
@@ -25,17 +25,52 @@
 #include "mega/waiter.h"
 #include <mutex>
 
+#ifndef USE_POLL
+    #define MEGA_FD_ZERO FD_ZERO
+    #define MEGA_FD_SET FD_SET
+    #define MEGA_FD_ISSET FD_ISSET
+    #define MEGA_FD_SET_T fd_set
```

Comment:
This one is not a macro, but a datatype. Perhaps we should use a `typedef`, and a naming in lowcase: `mega_fd_set_t` (or simply `fd_set_t`). Same for the definition if `USE_POLL`

Reply:
agree. changed in new commit